bms_victron_smart_shunt.orogen

```diff
@@ -15,9 +15,6 @@ import_types_from "power_base"
 task_context "Task", subclasses: "iodrivers_base::Task" do
     needs_configuration
 
-    # The number of packets needed to compose a full feedback update
-    property "packets_to_compose_a_full_feedback", "int", 2
```

Comment:
what would be the difference of just setting this property to 1(default or yml) instead of removing it entirely? Is this property useless, where any feedback that needs more than 1 packet invalid feedback?

Reply:
The number of necessary packet to compose a full feedback is unknown.
There is no information in the user manual and looks like the way that the device sends the info at the packets is not consistent.
So we decided to update the feedback upon every packet arrival.
This way this property is useless and doesn't make sense to be here. 1 is not the number of packets to compose a full feedback